### PR TITLE
Add page for `OpenNeuroDataShowcase`

### PR DESCRIPTION
**Pull request by weiglszonja**

### CODE REVIEW

File: HCK16_2023_OpenNeuroDataShowcase/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+* [Dates and Location](#dates-and-location)
+* [Objective](#objective)
+* [Eligibility](#eligibility)
+* [Application](#application)
+* [Organizing Committee](#organizing-committee)
+* [Disclaimer](#disclaimer)
+
+## Dates and Location
+
+- **Dates:** August 28, 9AM (PT) 2023
```
Comment (weiglszonja) [inline HCK16_2023_OpenNeuroDataShowcase/README.md:15]: TODO: Confirm dates 
@bendichter

File: HCK16_2023_OpenNeuroDataShowcase/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+Sign up to receive instructions to join Open NeuroData Showcase 2023 [here](https://forms.gle/TaiHL7fHjH7J9Een8). 
+
+# Virtual Data Showcase Agenda
+
+| Session                                                                                            | Speakers          | Time: Pacific |
+|----------------------------------------------------------------------------------------------------|-------------------|---------------|
+| Gather space opens to participants                                                                 |                   | 8:55am        |
+| Introduction to the virtual data showcase                                                          | Organizers        | 9:00-9:20am   |
+| Virtual poster session                                                                             | Data Contributors | 9:20-10:30am  |
```
Comment (weiglszonja) [inline HCK16_2023_OpenNeuroDataShowcase/README.md:45]: @bendichter I saw pdfs on the other pages, if you send me a source I can copy the contents of this to it and add it as pdf too.

File: HCK16_2023_OpenNeuroDataShowcase/README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+## Organizing Committee
+
+**Program chairs:**
+* Benjamin Dichter, CatalystNeuro
+* Oliver Rübel, Lawrence Berkeley National Laboratory
+* Ryan Ly, Lawrence Berkeley National Laboratory
+* Stephanie Albin, The Kavli Foundation
```
Comment (weiglszonja) [inline HCK16_2023_OpenNeuroDataShowcase/README.md:53]: copied this over from `NeuroDataRehack`

Comment (bendichter) [reply HCK16_2023_OpenNeuroDataShowcase/README.md:54]: put yourself on!

File: HCK16_2023_OpenNeuroDataShowcase/README.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+| Session                                                                                            | Speakers          | Time: Pacific |
+|----------------------------------------------------------------------------------------------------|-------------------|---------------|
+| Gather space opens to participants                                                                 |                   | 8:55am        |
+| Introduction to the virtual data showcase                                                          | Organizers        | 9:00-9:20am   |
+| Virtual poster session                                                                             | Data Contributors | 9:20-10:30am  |
+
+## Organizing Committee
+
+**Program chairs:**
+* Benjamin Dichter, CatalystNeuro
```
Comment (weiglszonja) [inline HCK16_2023_OpenNeuroDataShowcase/README.md:51]:
```suggestion
* Benjamin Dichter, CatalystNeuro
* Anna Szonja Weigl, CatalystNeuro
```

File: HCK16_2023_OpenNeuroDataShowcase/README.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+This virtual event is open for anyone interested in neurophysiology data, including but not limited to:
+* Neuroscience graduate students, post-docs, and other researchers interested in using open data in their research
+* Educators who want to use real datasets in their lessons
+* Undergraduates and high school students interested in learning about neuroscience
+* Open science advocates.
+
+## Application
+
+Sign up to receive instructions to join Open NeuroData Showcase 2023 [here](https://forms.gle/TaiHL7fHjH7J9Een8).
```
Comment (weiglszonja) [inline HCK16_2023_OpenNeuroDataShowcase/README.md:38]: TODO: replace this to point to the general registration form for everyone (this one is pointing to presenter form)

File: HCK16_2023_OpenNeuroDataShowcase/README.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+This virtual event is open for anyone interested in neurophysiology data, including but not limited to:
+* Neuroscience graduate students, post-docs, and other researchers interested in using open data in their research
+* Educators who want to use real datasets in their lessons
+* Undergraduates and high school students interested in learning about neuroscience
+* Open science advocates.
+
+## Application
+
+Sign up to receive instructions to join Open Neurodata Showcase 2023 [here](https://forms.gle/dh8eArbw836TpLyq8).
```
Comment (rly) [inline HCK16_2023_OpenNeuroDataShowcase/README.md]: Can you add the link to present in the event as well?

File: HCK16_2023_OpenNeuroDataShowcase/README.md [hunk 7]
```diff
@@ -36,7 +36,9 @@ This virtual event is open to anyone interested in neurophysiology data, includi
 
 ## Application
 
-Register to attend the Open Neurodata Showcase 2023 [here](https://forms.gle/dh8eArbw836TpLyq8). 
+If you are interested in **attending** the event, register [here](https://forms.gle/dh8eArbw836TpLyq8). 
+
+If you are interested in **presenting** your dandiset(s), register as a presenter at Open Neurodata Showcase 2023 [here](https://forms.gle/neGrkBaWsJvDoVd18).
```
Comment (weiglszonja) [inline HCK16_2023_OpenNeuroDataShowcase/README.md:41]: @rly just did, thanks!

**Comment by weiglszonja** [comment]
@rly let me know if there is anything else to modify, it would be nice if this goes live today or tomorrow.

**Comment by rly** [comment]
Looks good to me too

**Comment by rly** [comment]
Apparently the Test GitHub Pages Builds / run (pull_request) GitHub Action doesn't work on forks. We can test this on the main branch.

**Pull request merged**